module2/Homework2-start/ColorPicker/ContentView.swift

```diff
@@ -0,0 +1,167 @@
+import SwiftUI
+
+struct ContentView: View {
+  @Environment(\.verticalSizeClass) var verticalSizeClass
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass
+
+  @State private var alertIsVisible: Bool = false
+  @State private var redColor: Double = Constants.defaultRedColor
+  @State private var greenColor: Double = Constants.defaultGreenColor
+  @State private var blueColor: Double = Constants.defaultBlueColor
+
+  // When the app first runs, the default for previewColor shows up as white and not these values. Why?
+  @State private var previewColor: Color =  Color(red: Constants.defaultRedColor, green: Constants.defaultGreenColor, blue: Constants.defaultBlueColor)
+  
+  var body: some View {
+    
+    ZStack {
```

Comment:
Since you're not actually wanting to use the `ZStack` layering, a better choice here would be `Group` https://developer.apple.com/documentation/swiftui/group